src/main/java/com/uvocab/api/auth/AuthService.java

```diff
@@ -0,0 +1,20 @@
+package com.uvocab.api.auth;
+
+import lombok.AllArgsConstructor;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class AuthService {
+
+  private static final String FIXED_TOKEN = "str_token";
+
+  private final AuthenticationManager authenticationManager;
```

Comment:
@Danilosimoes we need configure this bean ! as you can see the IDEA show to us:
![image](https://github.com/user-attachments/assets/4395dfe3-95ab-43cf-b42b-525cbc676e0c)

**how configure ? see in the link of main PR.**